### DISCUSSION TITLE
Jog wheel direction

### DISCUSSION THREAD

**Discussion by tonnic** [teeminus/NoTouchScreenFirmware#33] [Q&A] [answered]
Is there an easy way to modify the jog wheel direction?   If I rotate clockwise, the menu selector goes up and temperature goes down, but my own intuition would prefer clockwise to move the selection down and make the temperature selection go up.

**Comment by Sineos** [comment, marked as answer]
Just switch the two encoder pins

**Reply by tonnic** [reply]
yep, you rock, thanks!
